src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -38,6 +38,7 @@ export default function applicantInformation(schema, options) {
   const prefix = (options && options.isVeteran) ? 'veteran' : 'relative';
   const mergedOptions = _.assign(defaults(prefix), options);
   const { fields, required, labels } = mergedOptions;
+  const fileNumberProp = prefix === 'relative' ? 'relativeVaFileNumber' : 'vaFileNumber';
```

Comment:
We have an issue to refactor this page to use the veteranId field, so this will probably be changed shortly.

The 5495 was the only form I found that was using a file number and this shared page definition.